packages/widgets/src/commandpalette.ts

```diff
@@ -860,7 +860,7 @@ export namespace CommandPalette {
      */
     renderItemLabel(data: IItemRenderData): VirtualElement {
       let content = this.formatItemLabel(data);
-      return h.div({ className: 'lm-CommandPalette-itemLabel', "aria-label": `${content}`, 'id': `${content}`.toLowerCase().replace(/ /g, '') }, content);
+      return h.div({ className: 'lm-CommandPalette-itemLabel', 'aria-live': "assertive", "aria-label": `${content}`, 'id': `${content}`.toLowerCase().replace(/ /g, '') }, content);
```

Comment:
please consider `polite` as a default. we don't have any aria live features yet and i think we'd want to reserve `assertive` aria live for VERY specific scenarios to avoid interrupting the expected screen behaviors.

Reply:
I have changed the attribute to polite in my latest implementation.